pkgs/build-support/emacs/melpa.nix

```diff
@@ -96,17 +113,17 @@ genericBuild ({
         -L "$NIX_BUILD_TOP/package-build" \
         -l "$melpa2nix" \
         -f melpa2nix-build-package \
-        $ename $version $commit
+        $ename $melpaVersion $commit
```

Comment:
Does it really work? I am impressed!

Reply:
Not sure why you comment at this line.  But yes, the whole PR works.  You can try it yourself :)

Reply:
This was mostly a reaction of 50% skeptic plus 50% amazed.